src/webapp/index.html

```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <meta charset="UTF-8">
+        <title>Journey</title>
+        <link rel="stylesheet" href="style.css">
+        <link rel="stylesheet" href="navbar.css">
+        <script src="script.js"></script>
+        <script src="https://maps.googleapis.com/maps/api/js?key=AIzaSyBAvfDBMIvi0xYZhEmntRWXxm5tb-KHciw"></script>
+    </head>
+
+    <body onload="initMap();">
+        <header class="header">
+            <nav class="navbar">
+                <a href="#index" class="nav-name">Journey</a>
+            </nav>
+        </header>
+
+        <div class = "padding row">
+            <div id="map"></div>
+                <div class = "content-box">
```

Comment:
Nit: The previous div is closed, but this div is indented. This gives an impression for reader that content-box is nested in map when in reality it is not. It also suggests that if the rest of the document is formatted correctly then it probably has an extra </div> somewhere that isn't paired with a corresponding <div> tag.